src/template-strings/info.js

```diff
@@ -23,6 +23,8 @@ const infoMsg = {
 
   IntegrationCreate: 'Created integration on API \'{{apiPath}}\'',
 
+  IntegrationDelete: 'Deleted integration from API \'{{apiPath}}\'',
```

Comment:
Could you follow the format
`'Deleted integration \'{{integrationId}}\' on API \'{{apiPath}}\'',`